networkit/cpp/graph/test/GraphToolsGTest.cpp

```diff
@@ -101,6 +101,48 @@ TEST_P(GraphToolsGTest, testDensity) {
     }
 }
 
+TEST_P(GraphToolsGTest, testVolume) {
+    constexpr count n = 100;
+    constexpr double p = 0.1;
+
+    auto doTest = [&](const Graph &G) {
+        // Volume for graph G
+        const auto volume = GraphTools::volume(G);
+
+        // Volume for either directed/undirected
+        double mod = directed() ? 1.0 : 2.0;
+        double baseG = weighted() ? static_cast<double>(G.totalEdgeWeight())
```

Comment:
The type of `totalEdgeWeight` is `edgeweight` (double), you don't need to cast it to double.